content/faq/unable-to-stream.md

```diff
@@ -4,17 +4,17 @@ category: troubleshooting
 order: 2
 ---
 
-If you are having trouble streaming all LBRY content, there may be an issue with your connectivity to the LBRY network. Sometimes simply restarting LBRY completely could fix the problem - this is usually what you want to try first. Both PC and router firewall settings, as well as other applications which limit internet connectivity (i.e. Internet security applications), may prevent your app from successfully connecting to the LBRY network - you can try disabling these or allowing LBRY to bypass them. Other times, it may be a hosting issue with the particular piece of content you are trying to download - please report in #help on [Discord](https://chat.lbry.io).  
+If you are having trouble streaming any LBRY content, there may be an issue with your connectivity to the LBRY network. Sometimes simply restarting the LBRY app and daemon completely could fix the problem - this is usually what you want to try first. Both PC and router firewall settings, as well as other applications which limit internet connectivity (i.e. Antiviruses or Internet security applications), may prevent your app from successfully connecting to the LBRY network. You can try disabling these or allowing LBRY to bypass them. Other times, it may be a hosting issue with the particular piece of content you are trying to download. Please report in the #help channel on [Discord](https://chat.lbry.io).  
 
-Another common cause of this issue is the lack of access to port 4444 (UDP).  LBRY employs UPnP (if enabled) in order to forward this port on your router automatically. If UPnP is disabled, it must either be enabled, or the port must be forwarded in your router configuration.  We have also seen instances where some ISPs may block port 4444, and for LBRY to download properly, this port must be manually configured - see steps below.  
+Another common cause of this issue is the lack of access to UDP port 4444. LBRY employs UPnP (if enabled) in order to forward this port on your router automatically. If UPnP is disabled, it must either be enabled, or the port must be forwarded in your router configuration.  We have also seen instances where some ISPs may block UDP port 4444, and for LBRY to download content properly, this port must be manually configured - see steps below.  
 
 ### How to manually set the DHT (Content Network) port
 
-1. Shutdown LBRY and lbrynet-daemon (check running processes)
-2. Go to your [lbrynet folder](https://lbry.io/faq/lbry-directories)
-3. If you have a `daemon_settings.yml` file, add this line to it at the end(44444 is an example, can be changed): `dht_node_port: 44444`.
+1. Shutdown LBRY and lbrynet-daemon (check running processes on Task Manager.)
```

Comment:
On a mac it's called "Activity Monitor", and lots of options in linux.
Perhaps "(check running processes using Task Manager on Windows or Activity Monitor on Mac)"

Reply:
amended